### DISCUSSION TITLE
FEATURE request . Wasp in a box logic

### DISCUSSION THREAD

**Discussion by dvbit** [jseidl/magic-areas#226] [Ideas / Feature Requests]
Hi
Great component.
Along with it I am using this appdaemon app
https://github.com/wernerhp/appdaemon_wasp

I think it would really be a valuable addition to add it to M.A. :-)

**Comment by jseidl** [comment]
Interesting concept. How does it handle if you leave the door open all time? Just counts the motion sensor?

**Reply by dvbit** [reply]
Yes . Sorry 4 the delayed answer

**Comment by mrnerdhair** [comment]
I'm looking for the same logic.

**Comment by jseidl** [comment]
This is queued to be worked on Soon(TM)

**Reply by Nuuki9** [reply]
That's great news. I use this with a bedroom and it works extremely well.

**Comment by jseidl** [comment]
Good news!

![image](https://github.com/user-attachments/assets/f093ac1c-cd80-4217-9c4e-62b817bb9d28)

PR is out, still WIP! https://github.com/jseidl/hass-magic_areas/pull/487
